RacingGame/Common/src/main/java/racing/common/data/TileType.java

```diff
@@ -32,8 +32,13 @@
     /**
      * Start
      */
-    START(5, 0.0, "tiles/start.png", false);
-
+    START(5, 0.0, "tiles/start.png", false),
```

Comment:
Isn't start deprecated, with the new spawn fields filling out that funcitonality?